kqueen_ui/blueprints/ui/forms.py

```diff
@@ -9,12 +9,18 @@
 from kqueen_ui.utils.forms import FlaskExtendableForm
 from wtforms.validators import DataRequired, Email, EqualTo, Length
 
+import pytz
+
 
 class LoginForm(FlaskExtendableForm):
     username = StringField('Username', validators=[DataRequired()])
     password = PasswordField('Password', validators=[DataRequired()])
 
 
+class UserProfileForm(FlaskExtendableForm):
+    timezone = SelectField('Timezone', validators=[DataRequired()], choices=[(x, x) for x in pytz.common_timezones])
```

Comment:
I suggest to add difference with UTC together with timezone name:
UTC+0000 (sometimes I pick first-found local name but with the same time difference

Reply:
Well you shouldn't though honestly, because different countries have different approach to daylight saving time, so if you pick correct UTC offset, but different country, you may get incorrect time in certain parts of the year

Reply:
![screenshotfrom2018-04-0516-43-11](https://user-images.githubusercontent.com/4489907/38369778-2886e854-38f1-11e8-9890-dd33b5e15928.png)
Check out google calendar settings, I think we also should provide time differences. Is it hard to do?

Reply:
My comment was more about that there is no reason to, not about difficulty, but I can check